app/src/main/java/com/android/voyageur/model/place/GooglePlacesRepository.kt

```diff
@@ -140,6 +140,7 @@ class GooglePlacesRepository(private val placesClient: PlacesClient) : PlacesRep
       onFailure: (Exception) -> Unit
   ) {
     val request = FetchPlaceRequest.builder(placeId, advancedPlaceFields).build()
+    var num = 0
```

Comment:
I think for this class you could add some comments to better explain what you're doing.

Reply:
I will make sure to add some, thanks for pointing it out.